tests/functional/pocket/test_navigation.py

```diff
@@ -5,9 +5,10 @@
 import pytest
 from selenium.webdriver.common.by import By
 
-from pages.externalpages.pocket.about import AboutPage
+from pages.pocket.about import AboutPage
 
 
+@pytest.mark.skip(reason="Disabled until we have a Pocket-mode server to test against - see #11509")
```

Comment:
Also I know we've skipped these for now, but once pocket mode is building / deploying we might want to consider adding a `pocket` marker so these tests are only run when in the right mode.